src/components/touchables/TouchableOpacity.tsx

```diff
@@ -57,7 +58,7 @@ export default class TouchableOpacity extends Component {
         style={[
           style,
           {
-            opacity: this.opacity,
+            opacity: (this.opacity as unknown) as number, // TODO: fix this
```

Comment:
TODO.

Reply:
`this.opacity` is an `Animated.Value`, so I don't know how to correctly do it. Maybe the implementation is incorrect?

Reply:
Hard to say, I'll take a look at it after we finish web.